cs/src/Management/IdGeneration.cs

```diff
@@ -0,0 +1,26 @@
ll", "question", "girls", "oatmeal", "jewel", "jam", "rifle", "skirt", "cabbage", "nation", "rhythm", "language", "bait", "page", "current", "distance", "trick", "trip", "detail", "cream", "maid", "scissors", "zephyr", "government", "wren", "treatment", "cast", "taste", "vase", "desk", "advertisement", "brother", "wilderness", "nose", "door", "trucks", "suit", "icicle", "ice", "tooth", "scent", "heat", "burst", "mask", "coat", "vacation", "prose", "year", "experience", "walk", "writer", "look", "middle", "church", "cheese", "picture", "box", "frog", "drain", "pie", "line", "pet", "underwear", "rail", "hammer", "truck", "haircut", "carriage", "girl", "rice", "pencil", "carpenter", "army", "tail", "pets", "trains", "flower", "night", "store", "camera", "beginner", "vest", "cave", "adjustment", "loss", "street", "lake", "death", "grape", "hole", "bath", "theory", "reading", "writing", "bridge", "whistle", "seed", "flowers", "harmony", "quartz", "cows", "feeling", "scarf", "sticks", "bead", "test", "back", "wood", "digestion", "pigs", "hat", "fang", "spark", "car", "tent", "birds", "river", "unit", "doctor", "water", "minister", "action", "stretch", "name", "chalk", "quilt", "caption", "liquid", "act", "fairies", "bone", "trail", "sand", "paper", "thread", "board", "eyes", "sort", "calculator", "discovery", "fear", "jump", "daughter", "square", "addition", "support", "glass", "pump", "beef", "fact", "holiday", "rabbits", "waves", "grass", "care", "spiders", "ball", "wealth", "quiver", "party", "believe", "person", "planes", "fowl", "group", "furniture", "bed", "pot", "beds", "bike", "vessel", "bird", "sister", "battle", "zoo", "idea", "activity", "driving", "airplane", "drawer", "dogs", "business", "coach", "slave", "snakes", "snail", "ticket", "egg", "market", "linen", "sponge", "basket", "cough", "thought", "hydrant", "mother", "soup", "plants", "guitar", "connection", "ocean", "teeth", "growth", "trade", "tub", "pleasure", "nerve", "fall", "smell", "ship", "example", "wrench", "relation", "produce", "weather", "pest", "machine", "bedroom", "berry", "transport", "argument", "teaching", "minute", "giraffe", "deer", "summer", "interest", "impulse", "lip", "kittens", "office", "creator", "stage", "monkey", "passenger", "toad", "account", "nut", "bulb", "step", "stick", "channel", "string", "seashore", "jellyfish", "fish", "verse", "town", "laborer", "jail", "expert", "development", "property", "tax", "son", "mitten", "flock", "crib", "aftermath", "jeans", "position", "twist", "sidewalk", "winter", "dime", "power", "fire", "shirt", "stove", "mouth", "knife", "lumber", "wax", "station", "recess", "regret", "babies", "baseball", "lock", "mind", "profit", "van", "curtain", "celery", "exchange", "skin", "acoustics", "chance", "crate", "room", "memory", "rock", "scale", "wheel", "grade", "shock", "friend", "blow", "crayon", "lettuce", "basketball", "cart", "brass", "ear", "eggnog", "wind", "religion", "condition", "iron", "shelf", "cats", "use", "dinner" };
```

Comment:
Did you check for offensive terms here? How did you get these lists?

Reply:
updated to use shorter lists and added random chars at the end